Document product area and cross-area teams

choldgraf opened the pull request:
In a recent Product and Engineering meeting, we discussed that there is some uncertainty about what kind of responsibility and authority our project manager and product manager roles had with our teams. This PR tries to disambiguate this by explicitly defining the structure and responsibilities of these parts of 2i2c. Here are major changes:

- Defines a `product/` area of 2i2c, along with responsibilities.
- Defines our structure and responsibilities for our `operations` area, and lumps our "Project Manager" role in this area.
- Updates our **organizational structure** page to note that while we are organized into functional areas, we work as cross-functional teams with shared resources like backlogs and meetings.
- Rebrands "Major Projects" to "Managed JupyterHub Service" since this is the sole initiative that we currently have going. It sort-of addresses https://github.com/2i2c-org/team-compass/issues/549 but I'll let @sgibson91 decide if this is enough. I'm still not sure what is the right structure to use here, but this seems like an iterative improvement.

choldgraf commented:
I'd like a review from one of @damianavila , @yuvipanda , or @jmunroe since they were the ones primarily involved in our discussion about these roles and relationships between people.

sgibson91 commented:
> * Rebrands "Major Projects" to "Managed JupyterHub Service" since this is the sole initiative that we currently have going. It sort-of addresses [Make our 'Support Process' docs more findable #549](https://github.com/2i2c-org/team-compass/issues/549) but I'll let @sgibson91 decide if this is enough. I'm still not sure what is the right structure to use here, but this seems like an iterative improvement.

This is an ideal change in my opinion. For me, it's really important to find important links, such as our support process, quickly which usually means from the right hand side menu, and this info was buried before.

damianavila commented on operations/structure.md:
```diff
@@ -0,0 +1,45 @@
+
+## Responsibilities
+
+2i2c's Operatoins team is responsible for defining and overseeing our systems for work coordination, planning, and accountability.
```
```suggestion
2i2c's Operations team is responsible for defining and overseeing our systems for work coordination, planning, and accountability.
```

damianavila commented on operations/structure.md:
```diff
@@ -0,0 +1,45 @@
+# Structure and responsibilities
+
+This page describes the structure and roles of our Product Team.
```
```suggestion
This page describes the structure and roles of our Operations Team.
```

damianavila commented on operations/structure.md:
```diff
@@ -0,0 +1,45 @@
+
+- Translate strategic priorities and deliverables into actionable work items.
+- Ensure that our work items have a clear scope and path forward for completion.
+- Communicate with our Product and Engineering teams to align our teams on a plan.
```
You are mentioning Prod and Eng here, you foresee Operations only connected to those areas or other areas as well??

choldgraf replied:
```suggestion
- Communicate with teams (e.g. `Product and Engineering` or `Community and Partnerships`) to align team members on a plan.
```

damianavila commented on operations/structure.md:
```diff
@@ -0,0 +1,45 @@
+### Communicate progress and status to key stakeholders
+
+- Make our work transparent, discoverable, and understanding to internal and external stakeholders.
+- Communicate with internal and external stakeholders to share information about the status and progress of major work items.
```
I am not sure Operations should directly connect to the external stakeholders. Communications with external, I think, should happen through Community and Partnership areas.

choldgraf replied:
I was thinking about a project manager style role, and how they are often communicating the status of projects, writing posts about it, etc. For example - the communication you're having with Arnim around the GESIS work. Maybe we should make it clear that this is when there are projects with external collaborators?

damianavila replied:
Again, in an ideal world, the PM gets input from the community/prod side about the external request. In the interim, for instance, it is ok for me to have conversations with Arnim, but in the future, that conversation should regularly happen between the external collaborator and (Comm and Prod), IMHO. The PM should not be tasked to inform external parties, their focus is to push forward the project accordingly to directions from the external-facing team.

damianavila commented on operations/structure.md:
```diff
@@ -0,0 +1,45 @@
+
+## Membership
+
+Damian Avila currently serves as a part-time Project Manager for our Engineering and Product team.
```
After thinking more about this, I actually like a lot to see an operations area in the structure.
Can you "formally" add this area to the escalidraw organizational diagram?

damianavila replied:
I agree it should live under an operation branch/area in the future... for now, I guess we could have it floating between the second and third layer, to the right, maybe? Do we need actually a 3D escalidraw representation 😜 ???

damianavila commented on organization/structure.md:
```diff
@@ -5,15 +5,36 @@ This page is a work-in-progress as we define the major organizational areas of 2
+- **[Operations](../operations/structure.md)**: Oversees our systems for coordination, planning, and executing our work according to our goals.
 - **[Administration](../administration/css.md)**: Carry out ongoing organizational tasks that are needed to keep 2i2c running on a daily basis.
+
+## Cross-functional teams
```
I like the distinction between functional areas and teams.
Naming might make those confusing so I appreciate the extra documentation about the idea.

choldgraf commented:
@damianavila I believe I've addressed your suggestions above - the one remaining one is not related to this PR but to the diagram that we had discussed. Do you think there's anything else to change in this PR?

damianavila commented:
> Do you think there's anything else to change in this PR?

Left some additional comments...

choldgraf commented:
OK I believe that I've addressed your comments @damianavila - to the point about external communications, here's what I've done:

- I've noted that the Operations area should focus on _internal_ communications about project status etc.
- I've noted that the Product area should focus on _external_ communications about service/feature changes, as well as owning our external service documentation and communications (and should interface w/ `Community` for deeper tutorial-style docs, and `Partnerships` for more marketing-style material.

cc @jmunroe to make sure that is reasonable to him as well.

damianavila commented:
> OK I believe that I've addressed your comments @damianavila

I will look into the updates later, but thanks for pushing forward on the differentiation between internal/external comms.

choldgraf commented:
@damianavila would appreciate a review from you or others so we can get this merged in quickly :-)

damianavila reviewed (approved):
LGTM now. Let's merge it and iterate!

choldgraf commented:
yay, thanks @damianavila !
